pipeline/tasks/gapic_tasks.py

```diff
@@ -61,10 +61,7 @@ def _move_to(self, gapic_config_dir, gapic_api_yaml):
         elif len(gapic_api_yaml) == 0:
             raise ValueError(error_fmt + 'No location specified')
         conf_out = os.path.abspath(gapic_api_yaml[0])
-        if os.path.exists(conf_out):
-            raise ValueError(error_fmt + 'File already exists')
```

Comment:
Is it really necessary? How often are all three of these conditions satisfied:

1) Running in local mode
2) Re-generating an existing GAPIC config (rather than a new one)
3) You have unstaged changes (i.e. not `git add`'d) in your googleapis sandbox

IIUC, that's the only time this would be a problem. Maybe we can just treat this as a user expectation issue?